pkg/k8sclient/k8sclient.go

```diff
@@ -96,8 +96,17 @@ func (e *NoK8sNetworkError) Error() string { return string(e.message) }
 
 // SetNetworkStatus sets network status into Pod annotation
 func SetNetworkStatus(client *ClientInfo, k8sArgs *types.K8sArgs, netStatus []nettypes.NetworkStatus, conf *types.NetConf) error {
```

Comment:
I don't know why you make into two function SetNetworkStatus and SetNetworkStatusAnnotation. I believe SetNetworkStatus/SetNetworkStatusAnnotation is not so much used (less than 10 times), so I suppose we just make one function.

Reply:
One of them uses `k8sArgs`, the other doesn't. The one that uses k8s args calls the other one, passing the explicit values.

I do this in order to keep the current caller code (in `multus.go`) untouched.